src/KafkaFlow.Retry.API/AppBuilderExtensions.cs

```diff
@@ -18,34 +18,83 @@ public static class AppBuilderExtensions
                     .ApplicationServices
                     .GetService(typeof(IRetryDurableQueueRepositoryProvider)) as IRetryDurableQueueRepositoryProvider;
 
-            appBuilder.UseRetryEndpoints(retryDurableQueueRepositoryProvider);
+            appBuilder.UseRetryEndpoints(retryDurableQueueRepositoryProvider, string.Empty);
+
+            return appBuilder;
+        }
+
+        public static IApplicationBuilder UseKafkaFlowRetryEndpoints(
+           this IApplicationBuilder appBuilder,
+           string endpointPrefix
+       )
+        {
+            var retryDurableQueueRepositoryProvider =
+                appBuilder
+                    .ApplicationServices
+                    .GetService(typeof(IRetryDurableQueueRepositoryProvider)) as IRetryDurableQueueRepositoryProvider;
+
+            appBuilder.UseRetryEndpoints(retryDurableQueueRepositoryProvider, endpointPrefix);
 
             return appBuilder;
         }
 
         public static IApplicationBuilder UseRetryEndpoints(
                     this IApplicationBuilder appBuilder,
-            IRetryDurableQueueRepositoryProvider retryDurableQueueRepositoryProvider
+            IRetryDurableQueueRepositoryProvider retryDurableQueueRepositoryProvider,
+            string endpointPrefix
         )
         {
             appBuilder.UseMiddleware<RetryMiddleware>(
                 new GetItemsHandler(
                     retryDurableQueueRepositoryProvider,
                     new GetItemsRequestDtoReader(),
                     new GetItemsInputAdapter(),
-                    new GetItemsResponseDtoAdapter()));
+                    new GetItemsResponseDtoAdapter(),
+                    endpointPrefix));
+
+            appBuilder.UseMiddleware<RetryMiddleware>(
+                new PatchItemsHandler(
+                    retryDurableQueueRepositoryProvider,
+                    new UpdateItemsInputAdapter(),
+                    new UpdateItemsResponseDtoAdapter(),
+                    endpointPrefix));
+
+            appBuilder.UseMiddleware<RetryMiddleware>(
+                new PatchQueuesHandler(
+                    retryDurableQueueRepositoryProvider,
+                    new UpdateQueuesInputAdapter(),
+                    new UpdateQueuesResponseDtoAdapter(),
+                    endpointPrefix));
+
+            return appBuilder;
+        }
+
+        public static IApplicationBuilder UseRetryEndpoints(
+                   this IApplicationBuilder appBuilder,
+           IRetryDurableQueueRepositoryProvider retryDurableQueueRepositoryProvider
+       )
+        {
+            appBuilder.UseMiddleware<RetryMiddleware>(
```

Comment:
do not duplicate this code, just call the new method with the parameter `endpointPrefix`